packages/gasket-plugin-https/lib/index.d.ts

```diff
@@ -7,68 +7,74 @@ import type { TerminusOptions, HealthCheckError } from '@godaddy/terminus';
 
 declare module '@gasket/engine' {
   type BaseListenerConfig = {
-    port?: number,
-    host?: string,
-    timeout?: number,
-    handler?: Function
-  }
+    port?: number;
```

Comment:
semicolons here coming from an eslint rule?

Reply:
Not from an eslint rule, but from prettier. Double checked the typescriptlang.org/docs and semi-colons are what they are using as well.